docs/validators.md

```
@@ -0,0 +1,534 @@
+```
+
+**File: `mixed.en.yaml`**
+```yaml
+user_name: "Username"           # ✅ Good
+userEmail: "Email"              # 🚨 camelCase
+user-phone: "Phone"             # 🚨 kebab-case
+User.Address: "Address"         # 🚨 Mixed styles
+```
+
+### Console Output
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)